samtranslator/plugins/api/implicit_api_plugin.py

```diff
@@ -93,6 +93,19 @@ def _generate_implicit_api_resource(self) -> Dict[str, Any]:
         Helper function implemented by child classes that create a new implicit API resource
         """
 
+    def _add_tags_to_implicit_api_if_necessary(self, resource: SamResource, template: SamTemplate) -> None:
```

Comment:
Is there no cleaner way to do it? This feels very duct tape-y.

Reply:
It would be much cleaner if we can move `GlobalsPlugin` to be in front of the implicit api plugin. I tried it, but it caused backward incompatible issue.